Neos.Neos/Classes/Fusion/Cache/CacheFlushingStrategy.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\Neos\Fusion\Cache;
+
+enum CacheFlushingStrategy
+{
+    case IMMEDIATELY;
```

Comment:
From a language-nerd perspective, that should be `IMMEDIATE`, no?

Because it's the "`IMMEDIATE` flushing strategy", not the "`IMMEDIATELY` flushing strategy"

Reply:
Works for me too. Will change that immediate(ly).

Reply:
Done